src/components/structures/UserSettings.js

```diff
@@ -86,6 +86,13 @@ const SIMPLE_SETTINGS = [
     { id: "pinUnreadRooms" },
     { id: "showDeveloperTools" },
     { id: "mergeUsersByLocalpart" },
+    {
+        id: "lowBandwidth",
+        fn: () => {
+            PlatformPeg.get().reload();
```

Comment:
do we really need to reload to apply the setting? It seems kinda weird to say someone is entering low bandwidth mode and start the whole app up again.

Reply:
Yeah, is a bit weird but I don't think you can turn crypto off once you've called initCrypto()

Reply:
ah, right.